### ISSUE TITLE
pgo-deployer storage configuration is misleading

### ISSUE THREAD

**Issue by Thoro** [CrunchyData/postgres-operator#1586]
**Describe the bug**
The page https://access.crunchydata.com/documentation/postgres-operator/4.3.1/installation/configuration/ describe the various options for defining the storage, there are environment variables listed, one would assume that every storage option 1 - 9 actually support all variables, painfully I noticed that that's not the case.

For example only storage 5 -9 support the class property!

**To Reproduce**
Try to set a storage class on storage 1 - 4, it will not be added to the pgo.yaml file.

**Expected behavior**
Every storage configuration should support every storage variable.

**Screenshots**

**Please tell us about your environment:**

* Operating System: Centos 7
* Where is this running ( Local, Cloud Provider): Local
* Storage being used (NFS, Hostpath, Gluster, etc): NFS
* Container Image Tag: 4.3.0
* PostgreSQL Version: 12
* Platform (Docker, Kubernetes, OpenShift): kubernetes 
* Platform Version: 1.15.3

**Additional context**

**Comment by Thoro** [comment]
Something is really broken with the deployer:

I set these variables:

```
                  - name: STORAGE8_NAME
                    value: "nfs-storage"
                  - name: STORAGE8_ACCESS_MODE
                    value: ReadWriteMany
                  - name: STORAGE8_CLASS
                    value: zfs-no-backup
                  - name: STORAGE8_SIZE
                    value: "4G"
                  - name: STORAGE8_TYPE
                    value: "dynamic"
                  - name: STORAGE8_SUPPLEMENTAL_GROUPS
                    value: "65534"
                  - name: STORAGE9_NAME
                    value: "xfs.shared-ssd"
                  - name: STORAGE9_ACCESS_MODE
                    value: "ReadWriteOnce"
                  - name: STORAGE9_CLASS
                    value: xfs.shared-ssd
                  - name: STORAGE9_SIZE
                    value: "4G"
                  - name: STORAGE9_TYPE
                    value: "dynamic"
```

The resulting pgo.yaml looks like this:

```
nfs-storage\nBackrestStorage: nfs-storage\nStorage:\n  nfs-storage:\n    AccessMode:
    \ \n    Size:  4G\n    StorageType:  dynamic\n    StorageClass: zfs-no-backup\n
    \ xfs.shared-ssd:\n    AccessMode:  ReadWriteOnce\n    Size:  4G\n    StorageType:
    \ dynamic\n    StorageClass: xfs.shared-ssd\nPgo:\n  Audit:  false\n  PGOImagePrefix:
    \ registry.developers.crunchydata.com/crunchydata\n  PGOImageTag:  centos7-4.3.0\n"
```

Sorry for the weird. format that's what kubectl gives me, but you can clearly see that the AccessMode is not filled out, that leads to that error:

``` 
5-30 13:26:05.227246252 +0000 UTC m=+280.678095315] - (create cluster failure) clustername hippo workflow  error PersistentVolumeClaim \"hippo-wal\" is invalid: spec.accessModes: Unsupported value: \"\": supported values: \"ReadOnlyMany\", \"ReadWriteMany\", \"ReadWriteOnce\"" func="events.Publish()" file="events/eventing.go:49" version=4.3.0
```

**Comment by jkatz** [comment]
Thanks for reporting. This was fixed in #1494 and will be part of the next release.

Due to some typos, there were issues with using storage classes 1, 4, and 8. The others should work.

I apologize for the inconvenience.
